test/spec/geo/leaflet/leaflet-cartodb-vector-layer-group-view.spec.js

```diff
@@ -75,4 +76,22 @@ describe('src/geo/leaflet/leaflet-cartodb-vector-layer-group-view.js', function
       done();
     });
   });
+  it('should call setUrl when all named map styles have been added', function () {
+    var cartoDBLayer1 = new CartoDBLayer();
+    var cartoDBLayer2 = new CartoDBLayer();
+    L.CartoDBd3Layer.prototype.setUrl = jasmine.createSpy();
```

Comment:
And we can remove `var L = window.L` if we remove this, right?

If we have to spy on `setURL` I would do it on the view:

```
spyOn(view, setURL).and...
```